src/WorkOS.net/Services/UserManagement/Entities/Session.cs

```diff
@@ -0,0 +1,48 @@
+namespace WorkOS
+{
+    using System.Collections.Generic;
+    using System.Linq;
+    using Newtonsoft.Json;
+
+    /// <summary>
+    /// Contains information about a user session.
+    /// </summary>
+    public class Session
+    {
+        /// <summary>
+        /// The Session ID.
+        /// </summary>
+        [JsonProperty("id")]
+        public string Id { get; set; }
+
+        /// <summary>
+        /// The timestamp of when the session was created.
+        /// </summary>
+        [JsonProperty("created_at")]
+        public string CreatedAt { get; set; }
+
+        /// <summary>
+        /// The timestamp of when the session will expire.
+        /// </summary>
+        [JsonProperty("expires_at")]
+        public string ExpiresAt { get; set; }
+
+        /// <summary>
+        /// The token which can be used to verify the user's session.
+        /// </summary>
+        [JsonProperty("token")]
+        public string Token { get; set; }
+
+        /// <summary>
+        /// List of organizations with authorized access that the user is a member of.
+        /// </summary>
+        [JsonProperty("authorized_organizaitons")]
+        public AuthorizedOrganizations[] AuthorizedOrganizations { get; set; }
```

Comment:
Can we use `List` here as well?

```suggestion
        public List<AuthorizedOrganizations> AuthorizedOrganizations { get; set; }
```

In .NET we almost always want to use `List` over `Array`.